doc/architecture/decisions/0005-use-a-guid-as-installation-id.md

```diff
@@ -0,0 +1,28 @@
+# 5. Use a GUID as Installation ID
+
+Date: 2018-04-27
+
+## Status
+
+Accepted
+
+## Context
+
+When installing a new device in the Notification Hub an Installation ID has to be provide by the App. Notification Hub
+says nothing about the format of this field, just that it is a string. Complete documentation can be found 
+[here](https://msdn.microsoft.com/en-us/library/azure/mt621153.aspx).
+
+## Decision
+
+We decided to enforce the type of the Installation ID field to be a canonical GUID as described
+[here](https://en.wikipedia.org/wiki/Universally_unique_identifier).
```

Comment:
Do we have a decision also on the UUID version to use? Since we don't have access to the device MAC address we're probably talking about Version 4 UUIDs, right?